From f93cd560a8a11213074c309e600d190bcf58c07a Mon Sep 17 00:00:00 2001
From: Stephan Hartmann <stha09@googlemail.com>
Date: Sat, 30 Apr 2022 17:10:21 +0000
Subject: [PATCH] libstdc++: fix incomplete type of
 content::ContentBrowserClient

Destructor of std::unique_ptr in libstdc++ uses sizeof() which
requires full definition of content::WebContentsViewDelegate in
content::ContentBrowserClient.
---
 content/public/browser/content_browser_client.cc | 1 +
 1 file changed, 1 insertion(+)

diff --git a/content/public/browser/content_browser_client.cc b/content/public/browser/content_browser_client.cc
index b9e9efe..f602a49 100644
--- a/content/public/browser/content_browser_client.cc
+++ b/content/public/browser/content_browser_client.cc
@@ -45,6 +45,7 @@
 #include "content/public/browser/url_loader_request_interceptor.h"
 #include "content/public/browser/vpn_service_proxy.h"
 #include "content/public/browser/web_contents.h"
+#include "content/public/browser/web_contents_view_delegate.h"
 #include "content/public/common/alternative_error_page_override_info.mojom.h"
 #include "content/public/common/content_features.h"
 #include "content/public/common/url_utils.h"
-- 
2.35.1

